CSSS/Bootstrap.cs

```diff
@@ -228,9 +234,14 @@ private void ShowUsage()
             Console.WriteLine("Optional arguments:");
 
             // Help
-            Console.Write("  -h, --help:".PadRight(rightPadding));
+            Console.Write("  -h, --help:".PadRight(RIGHT_PADDING));
             Console.WriteLine("Shows this help message");
 
+            // Shutdown the computer
+            Console.Write("  --shutdown".PadRight(RIGHT_PADDING));

```

Comment:
Could you add a colon after `--shutdown`, so that it's the same as all of the other arguments?